tests/unit/mixins/route-test.js

```diff
@@ -308,3 +308,73 @@ test('it uses overridden params when loading more data', assert => {
   assert.ok(model.get('reachedInfinity'), 'Should reach infinity');
 
 });
+
+test('it uses bound params when loading more data', assert => {
+
+  assert.expect(11);
+
+  var RouteObject = Ember.Route.extend(RouteMixin, {
+    model() {
+      return this.infinityModel('item', {perPage: 1, startingPage: 1}, {category: 'feature'});
+    },
+
+    perPageParam: 'testPerPage',
```

Comment:
Hmmm, @kellyselden, assuming you meant 317-319, if I don't define a model, how would I test loading more data with the third option that 'infinityModel' would take to define the dynamic property? 
The idea is that the two option hashes to 'infinityModel' are merged before submitting a request and this ensures the second option is passed on to the model. I also needed to set the startingPage to 1 and perPage to 1 so that I can load at least twice observing that the dynamic part got updatede on the second request.